ios-drag-drop.js

```diff
@@ -316,7 +318,8 @@
   }
 
   function once(el, event, handler, context) {
-    if(context) handler = handler.bind(context)
+    if(context)
+      handler = handler.bind(context);
```

Comment:
Can you revert this please, or give it brackets? This style of if statement is very error prone when people implicitly assume JS has significant whitespace.